pyproject.toml

```diff
@@ -1,66 +1,74 @@
 [tool.poetry]
-name = "hydrolib"
-version = "0.2.0"
-description = "HYDROLIB tools for hydrodynamic and hydrological modelling workflows"
+name = "hydrolib-test"
+version = "0.6.0"
+description = "Python wrappers around D-HYDRO Suite."
 authors = ["Deltares"]
-maintainers = [
-    "Arthur van Dam <arthur.vandam@deltares.nl>",
-    "Ruud Hurkmans <r.hurkmans@hkv.nl>",
-    ]
-license = "LGPL"
+license = "MIT"
+packages = [
+    { include = "hydrolib"},
+]
+readme = "README.md"
+repository = "https://github.com/deltares/hydrolib-core"
+documentation = "https://deltares.github.io/HYDROLIB-core"
+homepage = "https://deltares.github.io/HYDROLIB-core"
+
+[tool.poetry.urls]
+"issue tracker" = "https://github.com/Deltares/HYDROLIB-core/issues"
```

Comment:
This license change is maybe also unintended? We've set hydrolib-core to MIT, and hydrolib to LGPL a few years ago.
If it needs change, let discuss it separately and not do it in this PR.
Actually, maybe the entire file was accidentally copied/combined with a hydrolib-core pyproject.toml.
If so, can you revert the entire file and only update the relevant parts?

Reply:
...so I reverted it as suggested.